### PR TITLE
More icons

### PR DESCRIPTION
**Pull request by nonnominandus**
Finished #305.
preview

There was already a reference to goodtime in the appFilter.xml but with a different activity name and without an icon in the icons folder, so I added the new activity name and my version of an icon for this.

For the ALDI life MUSIK app I reused the napster icon (switched the drawable to napster in the appFilter.xml) as was concluded in the icon request.

### CODE REVIEW

**Comment by nonnominandus** [comment]
Added 2 more icons for #306: 
preview2

And added new activity names for ProtonVPN and LibreTorrent

**Comment by nonnominandus** [comment]
Created the two missing icons for #296.
preview4

**Comment by nonnominandus** [comment]
I finished #74 by creating the missing icons, and retouched the lloyds bank icon (see far right).
preview8

**Comment by nonnominandus** [comment]
I created all icons for #254 (except for one missing from the turtle output) and added all activity names.
preview9

**Comment by nonnominandus** [comment]
preview10
I created all icons for #308. And added activity names for already existing icons.

**Comment by nonnominandus** [comment]
preview11
I created all icons for #309 and added all activity names as usual.

**Comment by nonnominandus** [comment]
preview12
Now these icons look like this

**Comment by dkanada** [comment]
Wow I have a lot of icons to generate for this release :laughing: you guys have all been busy.

**Comment by nonnominandus** [comment]
I find it very nice to draw icons while listening to my lectures (which are all done via digital calling of course) so I have been pretty productive indeed.

File: other/7speaking.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">
```
Comment (dkanada) [inline other/7speaking.svg:1]: I think the first character has to be a letter.

**Pull request merged**